app/viewmodels/ResultsPageSummary.scala

```diff
@@ -49,7 +53,23 @@ class ResultsPageSummary(from: LocalDate, to: LocalDate, day: Boolean = true)
         content = span(messages("cf.cash-account.detail.csv")),
         visuallyHiddenText = Some(messages("cf.cash-account.detail.csv-definition"))
       )))
+    )
+  }
 
+  private def cashTransactionsResultRowV2(dates: CashTransactionDates,
+                                          fullStop: Boolean,
+                                          day: Boolean): SummaryListRow = {
+    summaryListRow(
+      value = HtmlFormat.escape(
```

Comment:
Could you please make a separate method for the code that has been done for value like below?

  private def rowResultBasedOnPeriod(dates: CashTransactionDates, fullStop: Boolean, day: Boolean) = {
    HtmlFormat.escape(
      if (fullStop) {
        rowResult(dates, day)
      } else {
        rowResultWithoutFullStop(dates, day)
      }).toString()
  }